botfront/cypress/integration/nlu_models/nlu_models.spec.js

```diff
@@ -1,143 +1,75 @@
-            cy.visit(`/project/${this.bf_project_id}/nlu/models`);
-            cy.get('.new-model').click();
-            cy.get('#uniforms-0000-0001').type(modelName);
-            cy.get('#uniforms-0000-0002 > .search').click();
-            cy.get('#uniforms-0000-0002 > .search').type('fre');
-            cy.get('#uniforms-0000-0002')
-                .contains('French')
-                .click();
-            cy.get('#uniforms-0000-0004').type('some description');
-            // TODO add instance when tests do instances
-            // Save
-            cy.get('.form > .primary').click();
-        });
+        cy.get('[data-cy=model-selector]').click();
+        cy.get('[data-cy=model-selector] input').type('Fre');
+        cy.get('[data-cy=model-selector]').contains('French').should('not.exist');
     });
 
-    describe('Model duplication', function() {
-        it('Should duplicate a model', function() {
-            cy.visit(`/project/${this.bf_project_id}/nlu/models`);
-            cy.contains('French').click();
-            cy.get('[data-cy=nlu-model-card] [data-cy=duplicate-button]').click();
-            cy.get('[data-cy=confirm-popup]');
-            cy.get('[data-cy=duplicate-button]').click();
-            cy.get('[data-cy=confirm-popup]').should('not.exist');
-            cy.get('[data-cy=duplicate-button]').click();
-            cy.get('[data-cy=confirm-popup]').contains('No').click();
-            cy.get('[data-cy=nlu-model-card]').should('have.lengthOf', 1);
-            cy.get('[data-cy=duplicate-button]').click();
-            cy.get('[data-cy=confirm-popup]').contains('Yes').click();
-            cy.get('[data-cy=nlu-model-card]').should('have.lengthOf', 2);
-            deleteSecondModel();
-        });
-    });
-
-    describe('Model publishing', function() {
-        it('should publish and unpublish a model', function() {
-            function checkPublishingState(publishedCount, unPublishedCount) {
-                cy.get('[data-cy=online-model]').should('have.lengthOf', publishedCount);
-                cy.get('[data-cy=offline-model]').should('have.lengthOf', unPublishedCount);
-            }
-
-            cy.visit(`/project/${this.bf_project_id}/nlu/models`);
-            cy.contains('French').click();
-            checkPublishingState(0, 1);
-            
-            cy.get('[data-cy=offline-model]').click();
-            cy.get('[data-cy=confirm-popup]').contains('No').click();
-            cy.get('[data-cy=confirm-popup]').should('not.exist');
-            checkPublishingState(0, 1);
+    it('should NOT be able to delete the default model', function() {
+        cy.visit(`/project/${this.bf_project_id}/settings`);
+        cy.get('.project-default-language .ui > .search').click();
+        cy.get('.project-default-language input').type('English');
+        cy.get('.project-default-language').contains('English').click();
+        cy.get('.project-settings-menu-info').click();
```

Comment:
Should use [data-cy...] directives instead of classes if possible

Reply:
Will try to use data-cy